### ISSUE TITLE
Don't include autom4te.cache in tarball

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/Semigroups#482]
The GAP 4.9.1 package distribution includes these files:
```
semigroups-3.0.15/src/libsemigroups/autom4te.cache:
output.0  output.1  requests  traces.0  traces.1
```
They do not really belong into a source distribution, and are not necessary to build semigroups resp. libsemigroups.

**Comment by james-d-mitchell** [comment]
@fingolfin Thanks for the report, we'll remove them in the next release.

**Comment by james-d-mitchell** [comment]
Resolved in v3.0.17.
